### ISSUE TITLE
[Feature Request] Set repetitions for each card?

### ISSUE THREAD

**Issue by maxpei** [tosimplicity/anki-preview-slideshow#5]
Thank you very much for the nice work! I'm using the add-on to facilitate language learning. In such a case, cards often have audios and it would be great if users are allowed to repeat each card a number of times before moving to the next ones.

**Comment by tosimplicity** [comment]
hit "R" or click "Replay Video", to replay audio. Pause the slideshow if you'd like to.
or do you want anki to automatically play it several times?

**Comment by maxpei** [comment]
Yes, I was talking about Anki playing each card several times automatically. I can hit R or click "Replay Video" when I am sitting in front of the computer. But sometimes I just let the slideshow run and read aloud the cards when I am doing other things. That's when I feel automated repetitions could be really helpful.

**Comment by tosimplicity** [comment]
check version 0.6 out.

**Comment by maxpei** [comment]
Got it. Thanks a lot!
